ui.tests/test-module/specs/fileinput/fileinput.runtime.spec.js

```diff
@@ -187,16 +201,23 @@ describe("Form with File Input - Prefill & Submit tests", () => {
                     cy.stub(win, 'open'); // creating a stub to check file preview
                 }
             });
-            const fileName = "empty.pdf";
+
             // attach the file
-            cy.get(fileInput.selector).attachFile(fileName);
+            cy.get(fileInput.selector).attachFile(fileInput.fileNames);
+            if(fileInput.type == 'Multi Select')
+                cy.get(fileInput.selector).attachFile('sample2.txt');
 
             // check for successful attachment of file in the view
-            checkFileNamesInFileAttachmentView(fileInput.selector, ['empty.pdf']);
+            checkFileNamesInFileAttachmentView(fileInput.selector, fileInput.fileNames);
+            if(fileInput.type == 'Multi Select')
+                checkFileNamesInFileAttachmentView(fileInput.selector, ['sample2.txt']);
 
             // check preview of the file
             checkFilePreviewInFileAttachment(fileInput.selector)
 
+            if(fileInput.type == 'Multi Select')
+                deleteSelectedFiles(fileInput.selector, ['sample.txt', 'sample2.txt'])
```

Comment:
Can you also write a test case of multiple selection of file when form is prefilled ?

Reply:
https://github.com/adobe/aem-core-forms-components/blob/master/ui.tests/test-module/specs/fileinput/fileinput.runtime.spec.js#L132

Reply:
wrote